PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/User.java

```diff
@@ -60,6 +61,10 @@
     @Patchable
     private UserRole userRole;
 
+    @Column(name = "is_enabled")
+    @ColumnDefault("TRUE")
```

Comment:
This should be good for updating current users. But I'm not sure if it works. Have You checked that? In db boolean is represented as 0 or 1. I don't think that it works if you have "TRUE". I does not work when you run app from docker-compose. While creating user value in db is 0 and you cannot log in.

Reply:
column generated by hibernate has type bit(1) with default value 1 so it works